source/_integrations/qnap.markdown

```diff
@@ -7,116 +7,28 @@ ha_release: 0.38
 ha_iot_class: Local Polling
 ha_codeowners:
   - '@colinodell'
+  - '@cyr-ius'
 ha_domain: qnap
```

Comment:
- Missing `ha_config_flow`.

Does it have discovery now as well? In that case, that needs to be added as well.